### PR TITLE
Hw03

### PR DESCRIPTION
**Pull request by ncaliskan**
@lendijoy
@diegodulanto

### CODE REVIEW

File: hw03/hw03/hw03.Rmd [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Homework 03
```
Comment (diegodulanto) [inline hw03/hw03/hw03.Rmd:1]: Don't forget to add the title, author, date, and output!

File: hw03/hw03/hw03.Rmd [hunk 2]
```diff
@@ -84,10 +84,19 @@ green_datar %>%
   3. Rescale the variables so that they go from 0-100 instead of the original range.
      Name the recaled variables `*_pomp`.
 ```{r}
+green_data_pomp <- green_datar %>% 
+  mutate(across(c(green1:open10),
+         ~ recode(.x, "1"= 0,"2" = 25, "3" = 50, "4" = 75,"5"= 100),
```
Comment (diegodulanto) [inline hw03/hw03/hw03.Rmd:89]: For cleaner code, I recommend entering a new line for your list!

File: hw03/hw03/hw03.Rmd [hunk 3]
```diff
@@ -21,48 +21,72 @@ For your assignment, do the following.
...
+#                    "comp10",
+#                    "intel7",
+#                    "intel8",
+#                    "intel9",
+#                    "intel10",
+#                    "open7",
+#                    "open8",
+#                    "open9",
+#                    "open10")
+#
```
Comment (diegodulanto) [inline hw03/hw03/hw03.Rmd:47]: Nice job spacing out your code!

File: hw03/hw03/hw03.Rmd [hunk 4]
```diff
@@ -21,48 +21,72 @@ For your assignment, do the following.
...
+#                    "open10")
+# 
+# recoded_green_data <- fixed_green_data %>% 
+#   mutate(across(all_of(data_reversed),
+#                 ~ recode(.x, "1"= 5,"2" = 4,"4" = 2,"5"= 1 ),
+#                 .names = "{.col}r") %>% 
+#            select(recoded_green_data))
+
+
+#Restarting everything because I couldn't figure out what was wrong.
```
Comment (diegodulanto) [inline hw03/hw03/hw03.Rmd:55]: I appreciate your candor and efforts!

File: hw03/hw03/hw03.Rmd [hunk 5]
```diff
@@ -21,48 +21,72 @@ For your assignment, do the following.
...
      Recode variables as needed.
 ```{r}
-data %>% 
-  select(.id, id, green1:student)
+green_datar %>% 
+  rowwise() %>% 
+  mutate(green_tot = sum(c_across(green1:green5), na.rm = TRUE),
+         comp_tot = sum(c_across(comp1:comp10r), na.rm = TRUE),
+         intel_tot = sum(c_across(intel1:intel10r), na.rm = TRUE),
+         open_tot = sum(c_across(open1:open10r), na.rm = TRUE))
```
Comment (diegodulanto) [inline hw03/hw03/hw03.Rmd:81]: Good work!

**Review by diegodulanto — COMMENTED** [review]
Good work on your coding!

File: hw03/hw03/hw03.Rmd [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+     
+  2. Compute total scores for the four scales. 
+     Recode variables as needed.
+```{r}
+green_datar %>% 
+  rowwise() %>% 
+  mutate(green_tot = sum(c_across(green1:green5), na.rm = TRUE),
+         comp_tot = sum(c_across(comp1:comp10r), na.rm = TRUE),
+         intel_tot = sum(c_across(intel1:intel10r), na.rm = TRUE),
+         open_tot = sum(c_across(open1:open10r), na.rm = TRUE))
```
Comment (lendijoy) [inline hw03/hw03/hw03.Rmd:81]: Do the recoded values replace the original values? Using a ":" to indicate a range of variables only works if the variables are consecutive. Specifically, if your recoded values are put at the end of your data set, indicating "intel1:intel10r" would take "intel1" as the first variable and "intel10r" as the last and include all variables in between.

**Review by lendijoy — COMMENTED** [review]
----------------------------------------------------
  Topic                       | Excellent | Satisfactory | Needs Work |
  ----------------------------|-----|-----|-----|
  **Coding style**            | | :heavy_check_mark:  |  |
  **Coding strategy**         |  | :heavy_check_mark: |  |
  **Presentation: graphs**    | | :heavy_check_mark:  |  |
  **Presentation: tables**    |  | :heavy_check_mark:  |  |
  **Achievement, creativity** | | :heavy_check_mark:  |  |
  **Ease of access**         | | :heavy_check_mark: | |
  
Remarks:
  - Great start on the assignment!
  - Work on reducing redundancy in your code, if something doesn't work maybe get rid of it... unless you intend to troubleshoot :smile:.

**Pull request merged**